application/controllers/collection_instrument.py

```diff
@@ -1,19 +1,22 @@
 import logging
 import structlog
+import uuid
 
 from json import loads
 from application.controllers.cryptographer import Cryptographer
 from application.controllers.helper import validate_uuid
+from application.controllers.rabbit_helper import send_message_to_rabbitmq
 from application.controllers.service_helper import service_request
 from application.controllers.session_decorator import with_db_session
 from application.controllers.sql_queries import query_business_by_ru, query_exercise_by_id, query_instrument, \
     query_instrument_by_id, query_survey_by_id
+from application.exceptions import RasError
 from application.models.models import BusinessModel, ExerciseModel, InstrumentModel, SurveyModel
 
 
 log = structlog.wrap_logger(logging.getLogger(__name__))
 
-UPLOAD_SUCCESSFUL = 'The upload was successful'
+RABBIT_QUEUE_NAME = 'Seft.Instruments'
```

Comment:
What's the reasoning for defining this here rather than just passing to to the `send_message_to_rabbitmq` function? We could define it in a `__init__` method for the class maybe?

Reply:
@andrewmil mostly just so we're following existing definitions of constants in this repo.